include/spirv-tools/libspirv.hpp

```diff
@@ -173,7 +203,7 @@ class SpirvTools {
   bool Validate(const uint32_t* binary, size_t binary_size) const;
   // Like the previous overload, but takes an options object.
   bool Validate(const uint32_t* binary, size_t binary_size,
-                const ValidatorOptions& options) const;
+                spv_validator_options options) const;
```

Comment:
why not const ref here?

Reply:
```
./include/spirv-tools/libspirv.h:typedef spv_validator_options_t* spv_validator_options;
```

That type is actually a pointer.  That is why.  This is done to expose the name in the API, but still hid the implementation.

Reply:
Sure, but it could use the const pointer variant, right?  Not a big deal, though.

Reply:
Sure can could make it a const.  I'll make that change.